hafnian/tests/test_samples.py

```diff
@@ -25,9 +25,9 @@
     hafnian_sample_classical_state,
     torontonian_sample_classical_state,
 )
-from hafnian.quantum import gen_Qmat_from_graph
+from hafnian.quantum import gen_Qmat_from_graph, density_matrix_element
 
-np.random.seed(20)
+np.random.seed(137)
```

Comment:
How come you needed to change the seed here?

Reply:
I just like 137, it is a more quantum electrodynamical number ;)

Reply:
🤣

Reply:
@nquesada You have just given me my new favoite random number seed. 👍